### PR TITLE
cli: Add subparsers and arguments

### PR DESCRIPTION
**Pull request by rye**
Part of #5.

### CODE REVIEW

**Comment by rye** [comment]
As of now, I think the argument parsing is done, so I'm going to merge and then in subsequent PRs add the actual validation and application logic.

**Comment by drewvolz** [comment]
👀

File: memberboat/__main__.py [hunk 1]
```diff
@@ -13,31 +13,62 @@
 #   - --dry-run -- only check and print what would happen
 
 
-def get_installation_token(owner, repo):
-	integration_id = os.environ['GITHUB_APP_ID']
-	private_key_file = os.environ['GITHUB_APP_PRIVATE_KEY_FILE']
+def get_installation_token(owner, repo, integration_id, private_key):
+	private_key = private_key
```
Comment (drewvolz) [inline memberboat/__main__.py:17]: @rye is this an artifact from the previous implementation? Not that it hurts to keep it, but what are your thoughts on nixing it?

Comment (rye) [reply memberboat/__main__.py:17]: Ah yes, I meant to remove that line; it should go. I can do that.

Comment (rye) [reply memberboat/__main__.py:17]: Done in #11.

**Pull request merged**